### PR TITLE
Done: Added homework for lesson 7

### PR DESCRIPTION
**Pull request by max3koz**
Done: Added homework for lesson 7

### CODE REVIEW

File: lesson_07/homework_07.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+
+print(sum_of_2_nums(2, 3))
+
+
+print("\nTask03.\nНаписати функцію, яка розрахує середнє арифметичне списку чисел.")
+
+
+def middle_of_nums(nums_list: list) -> float:
+    return sum(nums_list)/len(nums_list)
```
Comment (DartBrovsky) [inline lesson_07/homework_07.py]: It is better to add here validation of empty list. For ex. -> if len(nums_list) == 0: return None for ex until we do not know Exceptions.

File: lesson_07/homework_07.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+
+print(middle_of_nums([1, 2, 3, 4, 5]))
+
+
+print("\nTask04.\nНаписати функцію, яка приймає рядок та повертає його у зворотному порядку.")
+
+
+def revers_list(new_string: str) -> str:
+    return new_string[::-1]
```
Comment (DartBrovsky) [inline lesson_07/homework_07.py]: On the last lection we have learned method for reversing any iterable element. It is more clear than slicing.

**Review by DartBrovsky — APPROVED** [review]
We are good with solution. Take a look to my small comments and you are able to merge.

File: lesson_07/homework_07.py (hunk 2 above)
Comment (max3koz) [reply lesson_07/homework_07.py]: Updated code using the building function "reversed" in the second commit.

File: lesson_07/homework_07.py (hunk 1 above)
Comment (max3koz) [reply lesson_07/homework_07.py]: I changed the code and added the validation of the empty list in the second commit.

**Pull request merged**